docs/template_plugin/backend/CMakeLists.txt

```diff
@@ -33,7 +33,8 @@ target_compile_definitions(interpreter_backend
 )
 target_link_libraries(interpreter_backend PRIVATE ngraph::builder ngraph::reference openvino::util)
 
-target_include_directories(interpreter_backend PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+target_include_directories(interpreter_backend PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+                                               PRIVATE $<BUILD_INTERFACE:$<TARGET_PROPERTY:inference_engine_transformations,INTERFACE_INCLUDE_DIRECTORIES>>)
```

Comment:
Why cannot we link `interpreter_backend` with `openvino::dev` instead of this private include dependency?

Reply:
thanks, done